src/components/store/AddCartButton/style.module.scss

```
@@ -0,0 +1,70 @@
+@use 'src/styles/vars.scss' as vars;
+
+.addCartGroup {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  p {
+    color: var(--theme-text-on-background-1);
+    font-size: vars.$general-heading-4;
+  }
+
+  .buttonRow {
+    align-items: flex-end;
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+  }
+
+  .quantityColumn {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    @media (max-width: vars.$breakpoint-md) {
+      gap: 0.75rem;
+    }
+
+    div > select {
+      font-size: 1rem;
+      font-size: 1.25rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 1.25rem;
+      line-height: 200%; /* 2.5rem */
```

Is there a reason why you have multiple `font-size`s and `line-height`s here?

```suggestion
      font-size: 1.25rem;
      font-style: normal;
      font-weight: 700;
      line-height: 2.5rem;
```